packages/nx-aws-cache/src/generators/init/generator.spec.ts

```diff
@@ -15,6 +17,11 @@ describe('init generator', () => {
     appTree = createTreeWithEmptyWorkspace();
   });
 
+  afterAll((done) => {
```

Comment:
I see what's problem here, `await` is missing. Please use this code in both tests:
```
await generator(appTree, options);
```
And remove this `afterAll` logic.